Tasks/NuGetInstaller/NuGetInstaller.ps1

```diff
@@ -62,15 +65,40 @@ if(!$nuGetPath)
     $nuGetPath = Get-ToolPath -Name 'NuGet.exe';
 }
 
+if (-not $nugetPath)
+{
+    throw (Get-LocalizedString -Key "Unable to locate {0}" -ArgumentList 'nuget.exe')
+}
+
 if($nuGetRestoreArgs)
 {
+    if($nuGetRestoreArgs.ToLowerInvariant().Contains("-configfile"))
+    {
+        Write-Warning "ConfigFile was passed as a command line parameter, which may be ignored in certain parts of this task. Please specify the config file in the build definition instead."
```

Comment:
Don't forget to retrieve all of the Write-Host/Error/Warning/throw strings from the localization helper, otherwise it's just going to be future work for someone to do before Update 2 ships. Don't localize Write-Verbose.